homeworks/ekaterina.sharova_rogerosha/PokemonProject/style.css

```diff
@@ -1,22 +1,127 @@
+.upper-header {
+  background-color: #333;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 1fr auto;
+  padding: 10px;
+  color: #333;
+  gap: 50px;
+  height: 80px;
+  justify-content: space-between;
+}
+
+.ma-logo {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
+
+.MAlogo {
+  width: 29.11px;
+  height: 29.95px;
+  margin-left: 53px;
+}
+
+.MA {
+  width: 147.68px;
+  height: 9.08px;
+  margin-left: 13.21px;
+  margin-top: 10px;
+}
+
+.navigation {
+  text-align: right;
+  display: flex;
+  align-items: center;
+}
+
+.navigation ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.navigation-item {
+  font-family: Roboto, sans-serif;
```

Comment:
Please remove the underline for these menu items. And markers for li
<img width="1128" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/1c6d0e24-3a7d-43cd-985a-5cfc7897b3b6">

Reply:
And fix these gaps 
<img width="1309" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/b5c355d1-2cbc-48f9-8231-131fd7d1b315">